lighthouse-core/gather/gather-runner.js

```diff
@@ -105,7 +106,9 @@ class GatherRunner {
   static async setupDriver(driver, options) {
     log.log('status', 'Initializing…');
     const resetStorage = !options.settings.disableStorageReset;
-    // Enable emulation based on settings
+    // In the devtools/extension case, we can't still be on the site while trying to clear state
+    // So we first navigate to a blank page, then apply our emulation & setup
+    await GatherRunner.loadBlank(driver);
```

Comment:
another consequence of moving this is that the benchmarkIndex will be computed while on the previous page which may have other junk interfering. If we can't preserve the order maybe add a note to revisit if it becomes a problem or file an issue to investigate?

Reply:
can I just move `baseArtifacts.BenchmarkIndex = await options.driver.getBenchmarkIndex();` to after the call to `setupDriver`?

Reply:
ideally we run it before we start registering all the in-page listeners and whatnot, but that might be better than on existing page 👍